grid/chat/threads/maps/threads.ts

```diff
@@ -4,7 +4,7 @@ import { RecordingProcessOptions, SuperfaceTest } from '@superfaceai/testing';
 
 export const getThreadsTest = (
   provider: string,
-  workspace: string,
+  workspaces: string[],
```

Comment:
I'd prefer an object with `valid` and `invalid`, but if you insist on using an array, at least type it as a tuple (I think it's like `[valid: string, invalid: string]`